app/index.js

```diff
@@ -1,11 +1,15 @@
 import hapiApollo from '@as-integrations/hapi'
 
-import mockServer from '../mocks/server.js'
 import { context } from './graphql/context.js'
 import { apolloServer } from './graphql/server.js'
 import { server } from './server.js'
 import { logger } from './utils/logger.js'
 
+let mockServer
+if (process.env.ENABLE_MOCK_SERVER) {
+  mockServer = await import('../mocks/server.js')
```

Comment:
Can this just be taken out of the prod code entirely? If this is just for local testing, it can be started as a separate process as needed

Reply:
agreed to open as seperate ticket